packages/teams-js/src/public/interfaces.ts

```diff
@@ -221,9 +251,13 @@ export interface SecondaryId {
  * @beta
  */
 export enum SecondaryM365ContentIdName {
+  /** OneDrive ID */
   DriveId = 'driveId',
+  /** Teams Group ID */
   GroupId = 'groupId',
+  /** SharePoint ID */
   SiteId = 'siteId',
+  /** A guid string defining user ID */
```

Comment:
Knowing that this is a GUID is definitely helpful, but as a developer I wouldn't be sure where to get that GUID from? I assume I don't just make it up. Is it supposed to come from AAD?

Reply:
All of these are references to IDs specified on this page: https://learn.microsoft.com/en-us/graph/api/resources/onedrive?view=graph-rest-1.0

I don't know if we can guarantee that userId is always a Guid (maybe we can, but that page doesn't seem to say). From the app developer's perspective I think they should probably think of these ids as unknowns that they just plug into graph rest calls:

![image](https://user-images.githubusercontent.com/18446904/234914860-20b949a5-c85b-449d-9fb7-072c9ef2ab2e.png)

Might be worth adding that link to the comment and saying that the ids should be used to make Graph REST calls, as outlined at the link. What do you think @AE-MS ?

Reply:
I think that would provide sufficient information for a developer to know what they can safely use the ID for.